lib/framework/view/bottom_nav_page_group.dart

```diff
@@ -232,20 +234,24 @@ class _BottomNavPageGroupState extends State<BottomNavPageGroup>
       );
     }
 
-    return EnsembleBottomAppBar(
-      selectedIndex: widget.selectedPage,
-      backgroundColor: Utils.getColor(widget.menu.styles?['backgroundColor']) ??
-          Colors.white,
-      height: Utils.optionalDouble(widget.menu.styles?['height'] ?? 60),
-      padding: widget.menu.styles?['padding'],
-      color: unselectedColor,
-      selectedColor: selectedColor,
-      notchedShape: const CircularNotchedRectangle(),
-      onTabSelected: controller.jumpToPage,
-      items: navItems,
-      isFloating: fabMenuItem != null,
-      floatingAlignment: floatingAlignment,
-      floatingMargin: floatingMargin,
+    return ClipRRect(
```

Comment:
There'll be requirements for margin. Why don't we wrap it inside a Container so we can add margin in addition to border radius.

Reply:
@vusters Can you review it again? I changed it to Container and added `margin` and refactored the `padding`, `borderRadius` properties